bin/run.sh

```diff
@@ -159,12 +198,32 @@ ssl_pem=$SCITRAN_RUNTIME_SSL_PEM
 paste.app_factory = api.api:app_factory
 EOF
 
-paster serve --reload $TEMP_INI_FILE
+  if [[ $START == "1" ]]; then
+    paster serve --daemon $PASTER_INI_FILE
+    echo "See paster.log for execution log"
+  elif [[ $DEV == "1" ]]; then
+    paster serve --reload $PASTER_INI_FILE
+  fi
+fi
+
+
+###### SHUTDOWN TASKS
+if [[ $STOP == "1" || $DEV == "1" ]]; then
+  if [[ $STOP == "1" ]]; then
+    if paster serve --stop-daemon > /dev/null; then
+      echo "Paster killed."
+    fi
+  fi
+
+  if [[ -s "$SCITRAN_PERSISTENT_DB_PATH/mongod.lock" ]]; then
+    MONGO_PID=`cat "$SCITRAN_PERSISTENT_DB_PATH/mongod.lock"`
+    kill $MONGO_PID
+    waitfor $MONGO_PID
```

Comment:
Builtin `wait` can only wait on child processes. Mongo and paster are no longer children when daemonized. This busy-loop handles both cases.